ios/ItaliaApp/Info.plist

```diff
@@ -47,7 +47,7 @@
     <key>LSSupportsOpeningDocumentsInPlace</key>
     <true/>
     <key>NFCReaderUsageDescription</key>
-    <string>NFC tag to read NDEF messages into the application</string>
+    <string>NFC tag to read CIE cards</string>
```

Comment:
```suggestion
    <string>NFC tag to read EIC cards</string>
```

Reply:
[EIC - Electronic Identity Card](https://www.esteri.it/en/servizi-consolari-e-visti/italiani-all-estero/documenti_di_viaggio/carta_identita/)

Reply:
Indeed, thanks.